file-upload/file-upload-scala-sbt/fileupload-impl/src/main/resources/routes

```diff
@@ -1,3 +0,0 @@
-
```

Comment:
This is the Scala version. Not a problem but probably should be deleted in https://github.com/lagom/lagom-recipes/pull/67 instead.